impl/src/main/java/com/sun/faces/config/processor/FacesConfigNamespaceContext.java

```diff
@@ -23,9 +23,19 @@
 
 class FacesConfigNamespaceContext implements NamespaceContext {
 
+    String defaultNamespace = null;
+
+    public FacesConfigNamespaceContext() {
```

Comment:
I think extending accessibility of that constructor (current (existing) default one isn't public) is not necessary. Given type is package-accessible anyway.